### PR TITLE
Gyatt

### PR DESCRIPTION
**Pull request by 0xerorn**
yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
-# example-typescript-mod
-example eaglerforge mod made using typescript
+# Example TypeScript Mod
+Example EaglerForge mod template.
 
-## how to compile?
-first, run `npm i` to download dependencies and then do `npm run build` and the single js file should be in a folder called "build"
+## How to build?
+WARNING : Bun is experimental on windows
```
Comment (leahanderson1) [inline README.md:5]: bun shouldn't be the only option

Comment (0xerorn) [reply README.md:5]: i agree

Comment (radmanplays) [reply README.md:5]: me too

Comment (0xerorn) [reply README.md:5]: > me too

what should we do tho im silly

**Review by radmanplays — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (leahanderson1) [reply README.md:5]: just have npm since bun is backwards compatible with npm if people want to use bun they can

**Review by radmanplays — APPROVED** [review]

**Pull request merged**